wp-cli-templates/block-register-php.mustache

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * {{ title }} Block.
+ *
+ * @since 7.x
+ *
+ * @package Jetpack
+ */
+
+jetpack_register_block(
+	'jetpack/{{ slug }}',
+	array( 'render_callback' => 'jetpack_{{ slug }}_block_load_assets' )
```

Comment:
Slug could contain `-` so maybe pass this through `str_replace( '-', '_', $slug )` ?

Reply:
Good catch!